src/elements/hx-error/index.spec.js

```diff
@@ -61,7 +61,11 @@ describe('<hx-error> component tests', () => {
                 const component = /** @type { HXErrorElement } */ await fixture(template);
                 const name = component.slot;
 
-                expect(name).to.be.equal('');
+                if ( name !== null ) {
+                    expect(name).to.be.equal('');
+                } else {
+                    expect(name).to.be.null; // IE11 and Legacy Edge
+                }
```

Comment:
This test was updated because IE11 and Legacy Edge (plus older modern browsers) do have a  "real" Shadow DOM.  That's one of the main reasons we have pollyfills.